main.yml

```diff
@@ -354,20 +392,21 @@ ingress:
 insights:
   title: Insights
   frontend:
-    title: Red Hat Insights
+    title: Insights
     sub_apps:
       - id: dashboard
         default: true
       - id: advisor
+      - id: drift
+      - id: inventory
       - id: vulnerability
       - id: compliance
-      - id: patch
-      - id: drift
       - id: policies
-      - id: inventory
-      - id: remediations
+      - id: patch
+      - id: subscriptions
+        section: business
       - id: registration
-      - id: ros
```

Comment:
@ryelo & @karelhala,
Any reason behind removing `ros` sub-app from here?

Also, please check https://github.com/RedHatInsights/cloud-services-config/pull/529/files#r618296645

currently, we are not able to access ROS app on QA/stable as well as on local dev setup.